chia/wallet/trading/trade_store.py

```diff
@@ -78,6 +108,28 @@ async def create(
                 )
             )
 
+            await conn.execute(
+                (
+                    "CREATE TABLE IF NOT EXISTS coin_of_interest_to_trade_record("
+                    " trade_id text,"
```

Comment:
I don't think we should create new tables where we store binary data as hex strings